src/client/components/EnvelopeImage.tsx

```diff
@@ -3,51 +3,73 @@ import React from 'react';
 
 interface Props {
   width?: string;
-  cssOverrides?: SerializedStyles;
+  cssOverrides?: SerializedStyles | SerializedStyles[];
```

Comment:
Yes, `cssOverrides` is the name that Source uses to allow Emotion styles passed in. We follow that convention too so that Source would find it easier to extract our components and it should be easier to understand what's going on.

Reply:
Great, I didn't particularly like how it was both an array and an object but if we have no control over it, so be it 🤷